contracts/RewardsDistributorDelegate.sol

```diff
@@ -106,7 +140,7 @@ contract RewardsDistributorDelegate is RewardsDistributorDelegateStorageV1, Expo
         // Make sure distributor is added
         bool distributorAdded = false;
         address[] memory distributors = comptroller.getRewardsDistributors();
-        for (uint256 i = 0; i < distributors.length; i++) if (distributors[i] == address(this)) distributorAdded = true; 
+        for (uint256 i = 0; i < distributors.length; i++) if (distributors[i] == address(this)) distributorAdded = true;
```

Comment:
Gas optimization for loop:

```
uint256 distributorsLen = distributors.length;
for (uint256 i = 0; i < distributorsLen; ++i) 
```

Reply:
is distributors.length really calculated multiple times? this code was not changed its from original comp